### DISCUSSION TITLE
How to use AlienFX Tools

### DISCUSSION THREAD

**Discussion by Ragy9x** [T-Troll/alienfx-tools#430] [General]
Is there a detailed "how to use" manual on how to use the AlienFX Tools? I would really start using it with my Alienware X15 R2(12700H, 3070Ti) but whatever I do in the app all my fans suddenly go 100% and I cannot switch it off :-( I would really like to read a guide on the app and setup based on someone else :-D

**Comment by T-Troll** [comment]
Short guides available by link from project readme (click at tool name), as well as .md files into .zip
Also, wiki have a beginner's guide (but it's about lights mostly).
